Header header
int32 num_cpu_cores
float32[] cpu_usage
float32[] cpu_user
float32[] cpu_nice
float32[] cpu_system
float32[] cpu_idle
